backend_jobs/ingestion_pipeline/requirements.txt

```diff
@@ -0,0 +1,4 @@
+apache-beam==2.24.0
```

Comment:
file would benefit from some doc (what is this for?)

Reply:
cant doc text:-)